cypress/integration/join-page.spec.ts

```diff
@@ -146,23 +146,66 @@ describe('The join page', () => {
         });
       });
 
+      context('Using space characters in the join code', () => {
+        for (const goodJcId of [
+          mockJoinCodeId.replace(/\B(?=([0-9]{3})+(?![0-9]))/g, ' '),
+          // NO-BREAK SPACE
+          mockJoinCodeId.replace(/\B(?=([0-9]{3})+(?![0-9]))/g, '\u00A0'),
+          // CHARACTER TABULATION
+          mockJoinCodeId.replace(/\B(?=([0-9]{3})+(?![0-9]))/g, '\u0009'),
+          // ZERO WIDTH NO-BREAK SPACE
+          mockJoinCodeId.replace(/\B(?=([0-9]{3})+(?![0-9]))/g, '\uFEFF'),
+          mockJoinCodeId.replace(/\B(?=([0-9])+(?![0-9]))/g, ' '),
+          `   ${mockJoinCodeId.replace(/\B(?=([0-9]{2})+(?![0-9]))/g, '   ')}   `,
+        ]) {
```

Comment:
Those are quite the regular expressions! :joy: I appreciate the fanciness, but do you think we could do something like …? (It's more boring, to be sure, but this way I don't have to remember what a negative lookbehind does :stuck_out_tongue_winking_eye:)

```suggestion
        for (const goodJcId of [
          mockJoinCodeId.match(/[0-9]{3}/g).join(' '),
          // NO-BREAK SPACE
          mockJoinCodeId.match(/[0-9]{3}/g).join('\u00A0'),
          // CHARACTER TABULATION
          mockJoinCodeId.match(/[0-9]{3}/g).join('\u0009'),
          // ZERO WIDTH NO-BREAK SPACE
          mockJoinCodeId.match(/[0-9]{3}/g).join('\uFEFF'),
          mockJoinCodeId.split('').join(' '),
          `   ${mockJoinCodeId.match(/[0-9]{2}/g).join('   ')}   `,
        ]) {
```